React-Native/en/1.2.5.md

```diff
@@ -0,0 +1,15 @@
+# [Success Criterion 1.2.5 - Audio Description (Prerecorded)](https://www.w3.org/WAI/WCAG21/Understanding/audio-description-prerecorded.html)
+
+React-Native-Video offers support for [switching audio tracks](https://github.com/react-native-video/react-native-video/blob/master/API.md#selectedaudiotrack). This allows you to offer users a way to switch to an audio description track.
```

Comment:
Maybe also hyperlink the package and indicate it is a 3rd party package for clarity? It's clear when I review all success criteria, but if you haven't seen those, you might not know it is a package.

The [React-Native-Video package](https://github.com/react-native-video/react-native-video) offers support [...]